base/dates/ranges.jl

```diff
@@ -4,10 +4,12 @@
 
 # Override default step; otherwise it would be Millisecond(1)
 Base.colon{T<:DateTime}(start::T, stop::T) = StepRange(start, Day(1), stop)
+Base.colon{T<:Time}(start::T, stop::T) = StepRange(start, Second(1), stop)
 
 # Given a start and end date, how many steps/periods are in between
 guess(a::DateTime,b::DateTime,c) = floor(Int64,(Int128(b) - Int128(a))/toms(c))
 guess(a::Date,b::Date,c) = Int64(div(Int64(b - a),days(c)))
+len(a::Time,b::Time,c) = Int64(div(Int64(b - a),tons(c)))
```

Comment:
It is, just like for other TimeTypes. We indeed want to allow seeing the difference between two TimeTypes, but their addition is was doesn't make sense semantically.